.env.sample

```
@@ -1 +1,6 @@
-PORT=3000
\ No newline at end of file
+PORT=8080
+DB_USER=""
+DB_PASSWORD=""
+DB_HOST=""
+DB_PORT=5432
+DB_NAME=""
```

```suggestion
DB_NAME=""

```
new line 추가하면 좋을 것 같습니다!

---

오 감사합니다
빈줄을 추가하면 오류가 생기는 프로그램들이 있군요 처음알았습니다
감사합니다!!!